experimental/dds/tree2/package.json

```diff
@@ -17,14 +17,15 @@
 	"scripts": {
 		"bench": "mocha --timeout 999999 --perfMode --parentProcess --fgrep @Benchmark --reporter @fluid-tools/benchmark/dist/MochaReporter.js",
 		"bench:profile": "mocha --v8-prof --timeout 999999 --perfMode --fgrep @Benchmark --reporter @fluid-tools/benchmark/dist/MochaReporter.js && node --prof-process isolate-0x*-v8.log > profile.txt && rm isolate-0x*-v8.log && cat profile.txt",
-		"build": "fluid-build . --task build",
+		"build": "npm run depcruise && fluid-build . --task build",
```

Comment:
I don't think this is sufficient. To test, run `pnpm run build:compile tree2` from the root of the repo. It won't run depcruiser. This means you won't get any CI protection, because CI won't run it.

You need to update the fluid-build config to either run this as a pre-req for the build task (see the tasks in fluidBuild.config.cjs) or you need to add a root level script that runs the depcruiser task in each apckage that has it, e.g. `pnpm run -r --no-sort --stream --no-bail depcruise`. The former is probably easier.